src/Graphics/src/Graphics/ICanvas.cs

```diff
@@ -3,54 +3,236 @@
 
 namespace Microsoft.Maui.Graphics
 {
+	/// <summary>
+	/// Represents a platform-agnostic canvas on which 2D graphics can be drawn using types from the <see cref="Microsoft.Maui.Graphics"/> namespace.
+	/// </summary>
 	public interface ICanvas
 	{
+		/// <summary>
+		/// Gets or sets a value that represents the scaling factor to scale the UI by. 
```

Comment:
Scale the UI? Maybe drawing or canvas operations?

Reply:
This comes from a ported conceptual doc: https://learn.microsoft.com/en-us/dotnet/maui/user-interface/graphics/?view=net-maui-8.0

I like the sound of "... scale the canvas operations"